ts/src/bitteam.ts

```diff
@@ -0,0 +1,2395 @@
+                        'trade/api/reset-password': 1, // not unified looks like a private endpoint
+                    },
+                },
+                'private': {
+                    'get': {
+                        'trade/api/address/wallets': 1, // not unified returns 401000
+                        'trade/api/ccxt/balance': 1,
+                        'trade/api/ccxt/order/{id}': 1,
+                        'trade/api/ccxt/ordersOfUser': 1,
+                        'trade/api/ccxt/tradesOfUser': 1,
+                        'trade/api/discount': 1, // not unified returns 401000
+                        'trade/api/discounts': 1, // not unified returns 401000
+                        'trade/api/forgot-2fa': 1, // not unified returns 401000
+                        'trade/api/get-options': 1, // not unified returns 401000
+                        'trade/api/get-session-status': 1, // not unified returns 401000
+                        'trade/api/get-user-data': 1, // not unified returns 401000
+                        'trade/api/ordersByUser': 1, // not unified returns 401000
+                        'trade/api/phone/checkValidation': 1, // not unified returns 401000
+                        'trade/api/rate-wallbtc': 1, // not unified returns 401000
+                        'trade/api/sumsub/create': 1, // not unified returns 401000
+                        'trade/api/sumsub/data': 1, // not unified returns 401000
+                        'trade/api/sumsub/obtain': 1, // not unified returns 401000
+                        'trade/api/sumsub/status': 1, // not unified returns 401000
+                        'trade/api/tradesByUser': 1, // not unified
+                        'trade/api/transactionsByUser': 1, // not unified returns 401000
+                        'trade/api/transactionsOfUser': 1, // fetchDepositsWithdrawals
+                        'trade/api/transaction/{id}': 1, // not unified returns 401000
+                        'trade/api/user/api-keys': 1, // not unified returns 401000
+                        'trade/api/user/stats': 1, // not unified returns 401000
+                        'trade/api/user/stats/all': 1, // not unified returns 401000
+                        'trade/api/user/stats/ref': 1, // not unified returns 401000
+                        'trade/api/user/stats/ref/table': 1, // not unified returns 401000
+                    },
+                    'post': {
+                        'trade/api/access-check': 1, // not unified returns 401000
+                        'trade/api/all-orders/cancel': 1, // not unified returns 401000
+                        'trade/api/before-transaction-email': 1, // not unified returns 401000
+                        'trade/api/ccxt/cancel-all-order': 1,
+                        'trade/api/ccxt/cancelorder': 1,
+                        'trade/api/ccxt/ordercreate': 1,
+                        'trade/api/confirm-email': 1, // not unified returns 401000
```

Comment:
> not unified returns
What are these comments about?

Reply:
These endpoints are for the internal usage of the exchange's admins (they are present in documentation, but don't work). We can delete them, I think.

Reply:
if it is impossible to use them through ccxt we can delete them yes